cpp/log/cluster_state_controller-inl.h

```diff
@@ -0,0 +1,203 @@
+#ifndef CERT_TRANS_LOG_CLUSTER_STATE_CONTROLLER_INL_H_
+#define CERT_TRANS_LOG_CLUSTER_STATE_CONTROLLER_INL_H_
+
+#include <functional>
+
+#include "log/cluster_state_controller.h"
+#include "proto/ct.pb.h"
+
```

Comment:
No need for an empty line here, though. The order is "our header", the system headers, and then the project headers, I figure the protobuf generated headers still count as "project headers".